src/WebComponent.scss

```diff
@@ -1,19 +1,19 @@
-@mixin loadFonts($paths...) {
+@mixin loadFonts($fontFamily, $paths...) {
   @each $family in $paths {
```

Comment:
this loop seems useless now

Reply:
tbh, I didn't see the point of the loop before this update either, it's probably been useless for a while. But sometimes I don't question stuff enough. Will try to do that more